ansible/playbooks/registration.yaml

```diff
@@ -97,6 +97,20 @@
         - not_registered_found
         - "(rcg.rc != 0) or (use_suseconnect | bool)"
 
+    - name: Add additional authenticated modules
+      ansible.builtin.command: SUSEConnect -p {{ item.key }} -r {{ item.value }}
+      register: result
+      until: result is succeeded
+      retries: 10
+      delay: 60
+      when:
+        - ansible_facts['distribution_major_version'] == "12"
```

Comment:
leave out
```
- ansible_facts['distribution_major_version'] == "12"
```

the task is generic for any kind of SCC channels.  Our handling of LTSS is specific for 12sp5 but Ansible Task code is totally agnostic
 about it